frameworks/helloworld/tests/test_placement.py

```diff
@@ -24,47 +24,55 @@ def configure_package(configure_security):
         sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
 
 
-def _escape_placement_for_1_9(options: dict) -> dict:
-    # 1.9 requires `\"` to be escaped to `\\\"`
-    # when submitting placement constraints
-    log.info(options)
-    if sdk_utils.dcos_version_at_least("1.10"):
-        log.info("DC/OS version >= 1.10")
-        return options
-
-    def escape_section_placement(section: str, options: dict) -> dict:
-        if section in options and "placement" in options[section]:
-            options[section]["placement"] = options[section]["placement"].replace('"', '\\"')
-            log.info("Escaping %s", section)
-
-        log.info(options)
-        return options
-
-    return escape_section_placement("hello", escape_section_placement("world", options))
+@pytest.mark.dcos_min_version("1.12")
+@pytest.mark.sanity
+def test_scheduler_task_placement_by_marathon():
```

Comment:
This is the only new method in this class. Rest of the diff in this file is restructuring (I reordered methods so that test_* comes first followed by helpers). Did some cleanup in `get_task_host` as well.